CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.5.0-beta.1
+
+## ✨ Features
+* GeolocateControl displays a `disabled` icon if user denies geolocation permission. [#8871](https://github.com/mapbox/mapbox-gl-js/pull/8871))
+* Add `mapboxgl.getRTLTextPluginStatus()` to query the current status of the `rtl-text-plugin` ([#8864](https://github.com/mapbox/mapbox-gl-js/pull/8864))
+* `hash` Map option can now be set as a string which sets the map hash to a custom query parameter. ([#8603](https://github.com/mapbox/mapbox-gl-js/pull/8603)) (h/t [SebCorbin](https://github.com/SebCorbin))
+
+## 🍏 Improvements
+* `icon-text-fit` now properly respects `text-writing-mode`:`vertical` ([#8835](https://github.com/mapbox/mapbox-gl-js/pull/8835))
```

Comment:
nit: not really sure if there's a rule here, but I have a slight preference for "Add support for `icon-text-fit` with vertical labels."

Also I'm thinking whether this is an improvement or bug fix...?

Reply:
Tis a bug fix